otelx/jaeger.go

```diff
@@ -50,6 +50,7 @@ func SetupJaeger(t *Tracer, tracerName string, c *Config) (trace.Tracer, error)
 		jaegerRemoteSampler := jaegerremote.New(
 			"jaegerremote",
 			jaegerremote.WithSamplingServerURL(samplingServerURL),
+			jaegerremote.WithInitialSampler(sdktrace.TraceIDRatioBased(c.Providers.Jaeger.Sampling.TraceIdRatio)),
```

Comment:
Can you add this to `otelx/config_test.go`, please?

Reply:
Done!